### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by SavvasStephanides** [open-sdg/open-sdg#562]
While running an accessibility test using `A11yMachine`, one error arose:

> Error: This button element does not have a name available to an accessibility API. Valid names are: title attribute, element content.
> Note codes: H91 #WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name

> #search-btn

> <button aria-label="Search" id="search-btn" type="submit"><i class="fa fa-search" aria-hi...

It's a simple change, just adding an attribute to the search button. Is it worth updating Open SDG?

**Comment by brockfanning** [comment]
Sure, it can't hurt. I would have though the `aria-label` would be enough, but I'm fine with adding `title` as well, or whatever is best.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning I think maybe we could close as no remaining issues with button after DAC test?

**Comment by brockfanning** [comment]
Agreed.
